src/hooks/useOptions.ts

```diff
@@ -14,9 +15,7 @@ export const useOptions = (fileName: Files, lang: string) => {
 
   React.useEffect(() => {
     axios
-      .get(
-        `https://static.openfoodfacts.org/data/taxonomies/${fileName}.full.json`,
-      )
+      .get(`${OFF_URL}/data/taxonomies/${fileName}.full.json`)
```

Comment:
Is the removal of `static.` intentional?

Reply:
nope. good catch!